### PR TITLE
Move basic test utilities to netstandard1.3

### PR DESCRIPTION
**Pull request by jaredpar**
Necessary pre-req for getting the VB tests running on CoreClr.

### CODE REVIEW

**Comment by jaredpar** [comment]
CC @dotnet/roslyn-compiler for review

File: src/Compilers/Test/Utilities/VisualBasic/BasicTestBase.vb [hunk 1]
```diff
@@ -333,6 +282,46 @@ Public MustInherit Class BasicTestBase
             useLatestFramework:=useLatestFramework)
     End Function
 
+    Friend Shadows Function CompileAndVerifyEx(
```
Comment (cston) [inline src/Compilers/Test/Utilities/VisualBasic/BasicTestBase.vb]: >Shadows [](start = 11, length = 7)

Is this shadowing another method?

Comment (jaredpar) [reply src/Compilers/Test/Utilities/VisualBasic/BasicTestBase.vb]: Hmm that commit got into this PR accidentally. I will remove.

File: src/Test/Utilities/Portable/Platform/Custom/OSVersion.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    public static class OSVersion
+    {
+        /// <summary>
+        /// True when the operating system is at least Windows version 8
+        /// </summary>
+        public static bool IsWin8 =>
+#if NET46 || NETCOREAPP2_0
+            System.Environment.OSVersion.Version.Build >= 9200;
+#else
+            throw new InvalidOperationException();
```
Comment (sharwell) [inline src/Test/Utilities/Portable/Platform/Custom/OSVersion.cs]: :bulb: `PlatformNotSupportedException`?

Comment (jaredpar) [reply src/Test/Utilities/Portable/Platform/Custom/OSVersion.cs]: We haven't been using that for our `#if` solutions. Generally InvalidOperationException and NotSupportedException. I will look into changing this holistically in the change I'm working on right now. This PR is a series of changes to get our tests running on CoreClr.

Comment (jaredpar) [reply src/Test/Utilities/Portable/Platform/Custom/OSVersion.cs]: https://github.com/dotnet/roslyn/projects/16#card-10698974

**Pull request merged**